### PR TITLE
Add white outline stroke to logo.png

### PR DESCRIPTION
**Pull request by ptrfrncsmrph**
**Description of the change**

This would replace the existing `logo.png` used in the `README` with a higher-resolution version that has a white stroke outline. The new `logo.png` was created using this file from the PS logo repo: https://github.com/purescript/logo/blob/master/PS_Logo_Final.svg.

I went with the thicker stroke (1px) out of [the options presented here](https://github.com/purescript/purescript/issues/4001#issuecomment-770234610), but I don't have a strong preference. The stroke is on the _outside_ of the path so is not perceptible against a white background as seen in **Screenshots**. The fill color is `#14161a` as specified in the [PureScript logo repository](https://github.com/purescript/logo).

Fixes: #4001

<details><summary><strong>Before</strong></summary>

Screen Shot 2021-01-30 at 12 10 01 PM
Screen Shot 2021-01-30 at 12 09 47 PM

</details>

<details><summary><strong>After</strong></summary>

Screen Shot 2021-01-30 at 12 05 22 PM
Screen Shot 2021-01-30 at 12 05 36 PM

</details>

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [x] Linked any existing issues or proposals that this pull request should close
- [x] Updated or added relevant documentation
- [x] Added a test for the contribution (if applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ Bugfixes:
 Fixes a bug where the transitive closure of a module's dependencies
 where included in the `depends` field in the output for `purs graph`.
 
+* Add white outline stroke to logo in README (#4003, @ptrfrncsmrph)
```
Comment (hdgarrood) [inline CHANGELOG.md:26]: Can you move this into the “other” section please? I think this section ought to be for bug fixes inside the actual compiler executable.

Comment (ptrfrncsmrph) [reply CHANGELOG.md:30]: Done

**Review by hdgarrood — APPROVED** [review]

**Comment by hdgarrood** [comment]
Thanks!

**Review by thomashoneyman — APPROVED** [review]

**Comment by hdgarrood** [comment]
Oh, actually, sorry, would you mind adding yourself to CONTRIBUTORS.md?

**Pull request merged**